pkg/logging/yaml_parser.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package logging
+
+type ReqLogConfig struct {
```

Comment:
Can we make sure this is also rendered in our docs somewhere (auto gen)

Reply:
https://github.com/thanos-io/thanos/blob/86be7c366bdd9bc47655f6232fbc9dd83d37bbbd/scripts/cfggen/main.go#L62

Reply:
Thanks for pointing out!